js/components/info-section.js

```diff
@@ -0,0 +1,27 @@
+import buildElement from "../utils/buildElement.js";
+
+export default class InfoSection extends HTMLElement {
```

Comment:
a little confused on this web component. What does this do outside of regular HTMLElement methodology? As far as I can see, you're only take a container element, clearing the contents of it, and creating new html to append to it on the fly. 

Let me know if I'm missing something

Reply:
Wasn't sure if it needed to be a component. It is a part of the DOM and it needs data. I can change it so that it is already in the DOM (similar to icon bar).

Reply:
we'll keep it for now if you haven't changed anything. as long as it's working we'll keep it rolling